modules/cluster-logging-updating-logging.adoc

```diff
@@ -149,7 +149,7 @@ elasticsearch-rollover-infra   */15 * * * *   False     0        <none>
 $ oc exec -c elasticsearch <any_es_pod_in_the_cluster> -- indices
 ----
 +
-You should see the `app-0000x`, `infra-0000x`, `audit-0000x`, `.security` indices.
+You should see the `app-00000x`, `infra-00000x`, `audit-00000x`, `.security` indices.
```

Comment:
```suggestion
Verify that the output includes the `app-00000x`, `infra-00000x`, `audit-00000x`, `.security` indices.
```
> ! should verb
Whenever possible, rewrite in a more direct way.

Reply:
Resolved. I updated several instances of "You should see" in the document as recommended.